Database/Corrections/cataQuestFixes.lua

```diff
@@ -1644,6 +1644,9 @@ function CataQuestFixes.Load()
             [questKeys.objectives] = {},
             [questKeys.triggerEnd] = {"Fire Trial Completed",{[zoneIDs.AZSHARA] = {{33.43,23.39}}}},
         },
+        [14304] = { -- Blood Theory
+            [questKeys.extraObjectives] = {{nil, Questie.ICON_TYPE_INTERACT, l10n("Collect 10 Blood-filled Leeches"), 0, {{"monster", 25412}}}},
```

Comment:
When adding a new string that is handed to `l10n()` then that string also needed to be added to the translations.

The extraObjective strings are stored here: https://github.com/Questie/Questie/tree/master/Localization/Translations/ExtraObjectives

Reply:
https://github.com/Questie/Questie/pull/6179/commits/28b6d2741f66c25cc2d4a83800c89fe404d98a47